src/groups/mwc/mwcex/mwcex_executor.h

```diff
@@ -295,6 +294,18 @@ class Executor_Box_SboImp {
     /// size of the on-stack buffer used to store the executor target.
     struct Dummy {
         void* d_padding[4];
+
+        bool
+        operator==(BSLS_ANNOTATION_UNUSED const Executor_Box_SboImp::Dummy&)
```

Comment:
I don't think you'll get warning while using `BSLS_ANNOTATION_UNUSED` if you remove the argument name (like done in this declaration).

Reply:
Also, the argument type can just be `const Dummy&`